imsim/stamp.py

```diff
@@ -158,13 +162,11 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
                 logger.debug('From: noise_var = %s, flux = %s',noise_var,self.realized_flux)
                 gsparams = galsim.GSParams(folding_threshold=folding_threshold)
 
-            airmass = galsim.config.ParseValue(config, 'airmass', base, float)[0]
-            rawSeeing = galsim.config.ParseValue(config, 'rawSeeing', base, float)[0]
-            band = galsim.config.ParseValue(config, 'band', base, str)[0]
-            psf = self.Kolmogorov_and_Gaussian_PSF(gsparams=gsparams,
-                                                   airmass=airmass,
-                                                   rawSeeing=rawSeeing,
-                                                   band=band)
+            opt = { 'airmass': float, 'rawSeeing': float, 'band': str }
```

Comment:
Could we do these checks with the others above?

Reply:
I don't know what you mean by checks.  These are the three kwargs we need for Kolmogorov_and_Gaussian_PSF.  So GetAllParams here returns precisely what we need at this point.

Reply:
Oh, I see what you mean.  The return value from GetAllParams here starts out the same as params above.  Yes, we can skip running GetAllParams again now at this point.